openpype/hosts/maya/api/lib.py

```diff
@@ -2307,6 +2308,8 @@ def get_frame_range(include_animation_range=False):
             animation_start -= int(handle_start)
             animation_end += int(handle_end)
 
+        frame_range["frameStart"] = animation_start
+        frame_range["frameEnd"] = animation_end
```

Comment:
I don't believe this is the correct fix. I'd actually expect the caller to use `animationStart` and `animationEnd` for whatever it'd want to set instead but I might be misinterpreting the intended behavior for most studios here.